### PR TITLE
End_of_day

### PR DESCRIPTION
**Pull request by JakeGilla-DL**
$113 Genericises the end of day card and folds it into the greater gui layer functionality.

### CODE REVIEW

File: resources/scenes/game_scenes/level_1.tres [hunk 1]
```diff
@@ -6,6 +6,6 @@
 [resource]
 script = ExtResource("2_mss3e")
 scene = ExtResource("1_ee3oc")
-scene_id = "levelTest"
-scene_is_pausable = true
+scene_id = "game_world"
+scene_is_pausable = false
```
Comment (SlowFaller) [inline resources/scenes/game_scenes/level_1.tres]: why this change? don't we want to be able to pause?

**Review by SlowFaller — CHANGES_REQUESTED** [review]

File: resources/scenes/game_scenes/level_1.tres (hunk 1 above)
Comment (JakeGilla-DL) [reply resources/scenes/game_scenes/level_1.tres]: accidental, fixed now

**Review by SlowFaller — APPROVED** [review]
